package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@internetarchive/modal-manager",
+  "version": "0.0.0-alpha.19",
+  "description": "A Modal Manager Web Component",
+  "author": "modal-manager",
```

Comment:
I usually just drop the author field since they're released from the team account.